charts/linkerd2/templates/web.yaml

```diff
@@ -72,7 +72,7 @@ spec:
         - -grafana-addr={{.Values.grafana.name}}.{{.Values.global.namespace}}.svc.{{.Values.global.clusterDomain}}:3000
         {{- end}}
         {{- if .Values.tracing.enabled }}
-        - -jaeger-addr={{.Values.tracing.jaeger.name}}.{{.Values.global.namespace}}.svc.{{.Values.global.clusterDomain}}:16686
+        - -jaeger-addr=linkerd-jaeger.{{.Values.global.namespace}}.svc.{{.Values.global.clusterDomain}}:16686
```

Comment:
Why hard-coding `linkerd-jaeger` if it's still declared in `values.yaml`?

Reply:
yeah but its a add-on level value and hence control-plane components can't actually know these values, especially through Helm?

Reply:
Ok, OTOH if `jaeger.name` gets overridden at the add-on level, it won't work with the web component because the names won't match, right? If so, it might make more sense to hard-code it at the add-on level as well (and the same would apply for the other addons names).

Reply:
Yeah, so should we remove the `jaeger.name` and `collector.name` fields in this PR itself?

Reply:
I think so, yeah.